packages/gatsby/src/utils/webpack.config.js

```diff
@@ -324,8 +324,8 @@ module.exports = async (
       // directory if you need to install a specific version of a module for a
       // part of your site.
       modulesDirectories: [
-        directoryPath(`node_modules`),
         `node_modules`,
+        directoryPath(`node_modules`),
         directoryPath(`node_modules`, `gatsby`, `node_modules`),
```

Comment:
can you fix this second line while we are here? it should be 
```
directoryPath(`node_modules`, `gatsby/node_modules`)
```

the second path segment is getting ignored